src/lib/rest/rest.cpp

```diff
@@ -1042,6 +1035,15 @@ static int connectionTreat
   }
 
   // 3. Finally, serve the request (unless an error has occurred)
+  
+  // ServicePath check is delayed to third MHD call 
```

Comment:
Yes, three similar cases:  url, content type and out format checks are made in first MHD call. I will move them to third MHD call